pkgs/development/python-modules/mediapipe/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, absl-py
+, attrs
+, autoPatchelfHook
+, buildPythonPackage
+, fetchPypi
+, fetchurl
+, flatbuffers
+, gtk2
+, matplotlib
+, numpy
+, opencv4
+, protobuf
+, python
+, pythonAtLeast
+, pythonOlder
+, pythonRelaxDepsHook
+}:
+
+buildPythonPackage rec {
+  pname = "mediapipe";
+  version = "0.9.2.1";
+  disabled = pythonOlder "3.8" || pythonAtLeast "3.11";
+  format = "wheel";
+
+  poseHeavy = fetchurl {
+    url = "https://storage.googleapis.com/mediapipe-assets/pose_landmark_heavy.tflite";
+    sha256 = "WeQtcbzUTL26vEGfD/dmhllf0mVBlWa9QAnvcD6o4f4=";
+  };
+
+  src =
```

Comment:
Please take a look at https://github.com/NixOS/nixpkgs/blob/c58e6fbf258df1572b535ac1868ec42faf7675dd/pkgs/development/python-modules/home-assistant-chip-core/default.nix#L32-L51 to make this work with multiple python versions and system

Reply:
I'm not quite sure how